packages/react/src/components/form/NativeSelect/NativeSelect.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { forwardRef } from 'react';
+import type { ForwardedRef, ReactNode, SelectHTMLAttributes } from 'react';
+import cn from 'classnames';
+import { PadlockLockedFillIcon } from '@navikt/aksel-icons';
+
+import { omit } from '../../../utils';
+import { ErrorMessage, Label, Paragraph } from '../../Typography';
+
+import classes from './NativeSelect.module.css';
+import utilityClasses from './../../../utils/utility.module.css';
+import { useNativeSelect } from './useNativeSelect';
+
+export type NativeSelectProps = {
+  /**
+   * Specifies whether the select box is disabled.
+   * @default false
+   * */
+  disabled?: boolean;
+  /**
+   * Label that appears over the select box. */
+  label?: string;
+  /**
+   * Visually hides `label` and `description` (still available for screen readers)
+   * @default false
+   * */
+  hideLabel?: boolean;
+  /** Set to true to enable multiple selection. */
+  multiple?: boolean;
+  /** The ID of the `select` element. This will be generated if not provided. */
+  id?: string;
+  /**
+   * Defines the size of the select.
+   * @default 'medium'
```

Comment:
Remove the ticks as its showing wrong value in props table.
![image](https://github.com/digdir/designsystem/assets/1070981/15831a44-8b42-4c0f-80fa-14c95b56c5dd)

```suggestion
   * @default medium
```